core/store/orm/schema.go

```diff
@@ -55,6 +55,7 @@ type ConfigSchema struct {
 	EthRPCDefaultBatchSize                    uint32          `env:"ETH_RPC_DEFAULT_BATCH_SIZE" default:"100"`
 	EthTxResendAfterThreshold                 time.Duration   `env:"ETH_TX_RESEND_AFTER_THRESHOLD"`
 	EthereumDisabled                          bool            `env:"ETH_DISABLED" default:"false"`
+	EthereumHTTPURL                           string          `env:"ETH_HTTP_URL"`
```

Comment:
```
	EthereumHTTPURL                           string          `env:"ETH_HTTP_URL"`
	EthereumSecondaryURL                      string          `env:"ETH_SECONDARY_URL" default:""`
	EthereumSecondaryURLs                     string          `env:"ETH_SECONDARY_URLS" default:""`
	EthereumURL                               string          `env:"ETH_URL" default:"ws://localhost:8546"`
```
This has become a UX problem. We need a single env var, not proliferating env vars which are virtually indistinguishable from one another to the node operator. Given the direction we're heading, we should prefer http always, and have a single env var with a list of eth_urls.

Reply:
I agree, in practice it's a bit tricky to implement - see proposal here: https://www.notion.so/chainlink/PROPOSAL-Multiple-primaries-eth-node-load-balancing-cbfdaac4433d4a628a34bc37f614b901

Basically this PR is stage 1 in a multistage rollout. Eventually we need a more general/flexible approach to specifying eth nodes, I think probably in the database with a GUI.

We cannot use the node with HTTP only because we need websocket for subscriptions. Listing the URLs is not good enough because if we want to round-robin we need to have a concept of multiple urls per node.